configs/x86_64_defconfig

```diff
@@ -57,7 +57,6 @@ BR2_PACKAGE_LIBSSH2_OPENSSL=y
 BR2_PACKAGE_LIBXCRYPT=y
 BR2_PACKAGE_LIBOPENSSL_BIN=y
 BR2_PACKAGE_LIBCURL_CURL=y
-BR2_PACKAGE_LIBMNL=y
```

Comment:
For this and minimal, please verify that mnl is selected by some other package, otherwise iproute2 will not build devlink for x86 targets, see 374d24a54 for details.

Reply:
G-R-E-A-T catch! Thanks! :heart: